src/Text/KirbyTags.php

```diff
@@ -22,7 +22,15 @@ class KirbyTags
 
     public static function parse(string $text = null, array $data = [], array $options = []): string
     {
-        return preg_replace_callback('!(?=[^\]])\([a-z0-9_-]+:.*?\)!is', function ($match) use ($data, $options) {
+        $regex = '!
+            (?=[^\]])               # positive lookahead that matches a group after the main expression without including ] in the result
```

Comment:
I'm not sure if we really need this

Reply:
I'm really sorry, but I can't really help with this one. @lukasbestle @distantnative ?